presidio-analyzer/tests/test_entity_recognizer.py

```diff
@@ -31,3 +31,75 @@ def test_index_finding():
         match, start, tokens, tokens_indices
     )
     assert index == 3
+
+
+def test_remove_duplicates():
+    # test same result with different score will return only the highest
+    arr = [
+        RecognizerResult(
+            start=0,
+            end=5,
+            score=0.1,
+            entity_type="x",
+            analysis_explanation=AnalysisExplanation(
+                recognizer="test",
+                original_score=0,
+                pattern_name="test",
+                pattern="test",
+                validation_result=None,
+            ),
+        ),
+        RecognizerResult(
+            start=0,
+            end=5,
+            score=0.5,
+            entity_type="x",
+            analysis_explanation=AnalysisExplanation(
+                recognizer="test",
+                original_score=0,
+                pattern_name="test",
+                pattern="test",
+                validation_result=None,
+            ),
+        ),
+    ]
+    results = EntityRecognizer.remove_duplicates(arr)
+    assert len(results) == 1
+    assert results[0].score == 0.5
+    # TODO: add more cases with bug:
```

Comment:
Do we have a ticket on this? a future todo or something?

Reply:
Yes on azdo